src/services/expenses.service.js

```diff
@@ -0,0 +1,81 @@
+const { Expense } = require('../models/Expense.model');
+const { generateRandomId } = require('../Random');
+
+const getAllExpenses = async () => {
+  return Expense.findAll();
+};
+
+const getById = async (id) => {
+  return Expense.findByPk(id);
+};
+
+const normalizeCategory = (categories) => {
+  return Array.isArray(categories) || !categories ? categories : [categories];
+};
+
+const filterExpensesById = (expenses, id) => {
```

Comment:
The function 'filterExpensesById' is comparing 'userId' with 'id' using strict equality. However, you are parsing 'id' to an integer using 'parseInt'. This could potentially lead to inaccurate results if the 'userId' in the expenses is not an integer. It would be better to ensure that both values are of the same type before comparison.